### DISCUSSION TITLE
Feedback from 2.0.0 Beta

### DISCUSSION THREAD

**Discussion by Dead2** [zlib-ng/zlib-ng#855] [General]
Please leave general feedback from Beta 1 here.
If you found a bug, create a new issue for it.

General feedback is telling us it works on CPU X, operating system Y with application Z.
Or telling us about smaller issues/annoyances that does not warrant a separate issue.

**Comment by joebonrichie** [comment]
Used it as a drop-in system zlib replacement in Solus over the past year, used to have some strange issues that were only resolved by switching back to the canonical zlib such as occasional git commit hash mismatches but those issues seem to have been resolved with the releases.

The real test will probably be the -m32 built library and whether it is rock solid in terms of Steam compatibility when using native runtime although haven't tested that extensively yet.

Had to use autotools instead of the cmake build system as our cmake macros were not being properly respected when building for the 32bit version but looks like that's fixed in master now.

When the 2.0.1/2.0.2 release comes along will try to evaluate with the team if it'll be wise to replace our zlib with zlib-ng in compat mode. Most likely safety rebuilds will be performed to avoid any weird compiler shenanigans though I haven't had to do that locally at all. I can't realistically test _every_ package that uses zlib but so far compatibility has been good!
